python/templates/Julia.jl.jinja2

```diff
@@ -0,0 +1,18 @@
+{% for include in includes_jl %}
+{{ include }}
+{% endfor %}
+
+mutable struct {{ class.bare_type }}
+{% for member in Members %}
+	{{member.name}}::{{member.julia_type }}
```

Comment:
```suggestion
	{{ member.name }}::{{ member.julia_type }}
```
This is not really necessary and also not written down anywhere, but for the other templates we tried to follow this style where we had spaces before/after the `{{`/`}}`. It is slightly more readable, and in languages that use `{}` it makes it slightly easier to spot what is a template replacement and what is not.